presentation/invalid/ttml2-prstn-font-embedding-family-invalid.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<tt xml:lang="" xmlns="http://www.w3.org/ns/ttml"
+    xmlns:ttm="http://www.w3.org/ns/ttml#metadata"
+    xmlns:ttp="http://www.w3.org/ns/ttml#parameter" ttp:profile="ttml2-presentation" ttp:validationAction="warn"
+    xmlns:tts="http://www.w3.org/ns/ttml#styling">
+  <head>
+    <metadata>
+      <ttm:desc>invalid embedded font family name - matches generic font family</ttm:desc>
+    </metadata>
+    <resources>
+      <font src="../resources/test.otf" family="serif"/>
+    </resources>
+    <styling>
+      <style xml:id="sBody" tts:fontFamily="serif" tts:fontSize="18px" tts:lineHeight="22px" tts:color="black" tts:textAlign="center"/>
+      <style xml:id="sBackground" tts:backgroundColor="#A0A0A0"/>
+    </styling>
+    <layout>
+      <region xml:id="area1" style="sBackground" tts:extent="480px 128px" tts:position="center" tts:displayAlign="center"/>
+    </layout>
+  </head>
+  <body region="area1" style="sBody">
+    <div>
```

Comment:
The presentation test result must use the default font and not the referenced font (ScheherazadeRegOT if my suggestion above is accepted). 

```suggestion
    <div>
       <metadata>
          <ttm:desc>Expected result: text should be presented in default font.</ttm:desc>
          <ttm:desc>Incorrect result: text is presented in ScheherazadeRegOT.</ttm:desc>
       </metadata>
```

Reply:
ok